src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -211,6 +211,25 @@ private void updateSuppressFromDiscoveryFlag(WebContext wContext, Instance updat
     }
   }
 
+  private void deleteSourceStorageRecord(WebContext wContext, String instanceId) {
+    try {
+      SourceStorageRecordsClient srsClient = getSourceStorageRecordsClient(wContext);
+      srsClient.deleteSourceStorageRecordsById(instanceId, INSTANCE_ID_TYPE, httpClientResponse -> {
+        if (httpClientResponse.result().statusCode() == HttpStatus.HTTP_NO_CONTENT.toInt()) {
+          log.info(format("Source storage record was successfully deleted in SRS. InstanceID: %s", instanceId));
```

Comment:
For the future, this could look like below
`log.info("Source storage record was successfully deleted in SRS. InstanceID: {}", instanceId);`